### ISSUE TITLE
Customizing dates which it pulls

### ISSUE THREAD

**Issue by w4rner** [cleverdevil/micromemories#7]
As [posted on Micro.blog](https://micro.blog/w4rner/11535922):

Idea: 
Love the "On this Day" page. Reminds me of the section in Cory Doctorow's Pluralistic. I'd love to know how! I'd implement a more recent exponential version: **1 day, 1 week, 1 month, 1 year ago**. Would keep me honest about posting every day :D

Technical
It would basically mean **tweaking items_for function in fetch.py** right?

**Comment by cleverdevil** [comment]
Yes, that's likely what would need to be changed. I don't have bandwidth to add new features, but will happily review any PRs.
